PLUGIN-GUIDE.md

```diff
@@ -152,6 +154,21 @@ Here is an example of a `package.json` file for a Leaflet plugin.
 }
 ```
 
+If possible, do not commit your minified files (e.g. `dist`) to a repo; this can
+lead to confussion when trying to debug the wrong file. Instead, let the user 
+run the build/minify scripts automatically when they run `npm install`, for example:
+
+```json
+{
+  "name": "my-leaflet-plugin",
+  ...
+  "scripts": {
+    "postinstall": "grunt build"
```

Comment:
Actually the current best practice is to build in `prepublish` rather than run build on user install, so that the files get published to NPM and are available on install without devDependencies immediately.